Frontend/src/app/components/research-lab/recency-chart/recency-launch-config/recency-launch-config.component.scss

```
@@ -0,0 +1,421 @@
+.launch-config__run-detail {
+  margin-top: 0.15rem !important;
+  color: $text-muted;
+  font-size: var(--fs-xs);
+}
+
+.sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)